tests/Helpers.elm

```diff
@@ -1,21 +1,81 @@
 module Helpers exposing
-    ( areStatements
+    ( app
+    , areStatements
+    , areStatementsSansMeta
+    , binOp
+    , case_
     , fails
-    , isApplication
+    , fakeMeta
+    , integer
+    , isApplicationSansMeta
     , isExpression
+    , isExpressionSansMeta
     , isStatement
+    , isStatementSansMeta
+    , list
+    , record
+    , recordUpdate
+    , tuple
     , var
     )
 
 import Ast exposing (parse, parseExpression, parseStatement)
 import Ast.BinOp exposing (operators)
-import Ast.Expression exposing (Expression(..))
-import Ast.Statement exposing (ExportSet(..), Statement(..), Type(..))
+import Ast.Expression exposing (Expression(..), ExpressionSansMeta(..), MExp, dropExpressionMeta, dropMExpMeta)
+import Ast.Helpers exposing (WithMeta)
+import Ast.Statement exposing (ExportSet(..), Statement(..), Type(..), dropStatementMeta)
 import Expect exposing (..)
 
 
-var a =
-    Variable [ a ]
+
+-- Structures
+
+app : MExp -> MExp -> MExp
+app left right =
+    fakeMeta (Application left right)
+
+
+record =
+    fakeMeta << Record
+
+
+recordUpdate : String -> List ( WithMeta String, MExp ) -> MExp
+recordUpdate name =
+    fakeMeta << RecordUpdate (fakeMeta name)
+
+
+var : String -> MExp
+var name =
+    mvar 0 0 name
+
+
+integer =
+    fakeMeta << Integer
+
+
+binOp name l r =
+    fakeMeta <| BinOp name l r
+
+
+tuple =
+    fakeMeta << Tuple
+
+
+case_ mexp cases =
+    fakeMeta <| Case mexp cases
+
+
+list =
+    fakeMeta << List
+
+
+mvar : Int -> Int -> String -> MExp
+mvar line column name =
+    { meta = { line = line, column = column }, e = Variable [ name ] }
+
+
+
+-- Helpers
```

Comment:
I believe it's better to use them instead of raw `ExpressionSansMeta` constructors since if we change our minds or naming or whatever we won't have to change the tests, just the helpers.